libs/blocks/marquee/marquee.js

```diff
@@ -40,7 +40,7 @@ const decorateBlockBg = (block, node) => {
     if (childCount === 3) {
       child.classList.add(viewports[index]);
     }
-    const videoElement = child.querySelector('a[href$=".mp4"], video source[src$=".mp4"]');
```

Comment:
I think this is here due to a race condition.

If the video block loads before marquee, you will have a video element and not just a link.

Reply:
This is correct.

Reply:
Maybe address this in the dup func in this block decorateVid otherwise big picture we should refactor this to use a shared video decorator and import it rather. 
```
  const isAutoplay = !!(hash?.includes('autoplay'));
  const isNotLooped = !!(hash?.includes('autoplay1'));

  const attrs = isAutoplay ? `playsinline autoplay ${isNotLooped ? '' : 'loop'} muted` : 'playsinline controls';
  const video = `<video ${attrs}>
        <source src=".${pathname}" type="video/mp4" />
      </video>`;